tasks/request_files/request_files.py

```diff
@@ -430,6 +430,18 @@ def process_granule(
                         err=err,
                     )
                     a_file[FILE_ERROR_MESSAGE_KEY] = str(err)
+                except KeyError as err:
```

Comment:
Could mess with the existing flow to make this a non-retry error, but didn't feel like a good idea with 336 already planned.

Reply:
Done

Reply:
See comments below about duplicate error logging.